### PR TITLE
Add Twilio Runtime node forward call example

### PR DESCRIPTION
**Pull request by stefanjudis**
After getting [a Twilio Runtime template merged](https://github.com/serverless/serverless/pull/6467) into the serverless CLI I'd love to add an example here, too. :)

### CODE REVIEW

File: twilio-node-forward-call/package.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "name": "twilio.node-forward-call",
+  "version": "1.0.-",
+  "description": "Example demonstrating Twilio Runtime support with an endpoint return TwiML to forward a phone call",
```
Comment (ScottBrenner) [inline twilio-node-forward-call/package.json]:
```suggestion
  "description": "Example demonstrating Twilio Runtime support with an endpoint that returns TwiML to forward a phone call",
```

**Comment by stefanjudis** [comment]
Ooops. @ScottBrenner thanks for the typo fix. :)

**Comment by eahefnawy** [comment]
Thanks @stefanjudis ... the build is failing though, I think you need to regenerate the README. Once that is done I can merge right away. 🎉

**Comment by stefanjudis** [comment]
@eahefnawy Updated the readme. 🤞

**Comment by stefanjudis** [comment]
Thanks a bunch @eahefnawy. 🙇‍♂️

**Pull request merged**